### PR TITLE
feat: Make collection name customizable

### LINKED ISSUE #2: Make collection name customizable

### PR DESCRIPTION
**Pull request by dantehemerson**
Fix #2

### CODE REVIEW

**Comment by dantehemerson** [comment]
@qqilihq I have problems running tests. I think it's not matching files, I tried changing **jest config** without success. 
Maybe I'm doing something wrong? 
image

**Comment by qqilihq** [comment]
Mmmmh, couldn’t reproduce the trouble with the tests. On the CI environment and locally (running macOS) this works fine. And the test also pass:

```
$ yarn test
yarn run v1.16.0
$ jest
 PASS  test/index.test.ts
  migrate MongoDB state store
    initialization
      ✓ can be instantiated (6ms)
    errors
      ✓ throws error when connection fails (7ms)
    loading state
      ✓ throws error when migrations collection contains more than one document (10ms)
      ✓ returns empty object when migration collection is empty (14ms)
      ✓ returns migrations document (4ms)
    saving state
      ✓ inserts new document into empty migrations collection (4ms)
      ✓ replaces existing document in migrations collection (3ms)

  console.log
    No migrations found, probably running the very first time

      at lib/index.ts:751:17

----------|---------|----------|---------|---------|-------------------
File      | % Stmts | % Branch | % Funcs | % Lines | Uncovered Line #s 
----------|---------|----------|---------|---------|-------------------
All files |     100 |    83.33 |     100 |     100 |                   
 index.ts |     100 |    83.33 |     100 |     100 | 9-10              
----------|---------|----------|---------|---------|-------------------
Test Suites: 1 passed, 1 total
Tests:       7 passed, 7 total
Snapshots:   0 total
Time:        4.118s
Ran all test suites.
✨  Done in 14.43s.
```

Not sure what the trouble might be?

Only one minor suggestion: Could you move the `Options` interface directly into the `index.ts` to avoid an additional `import` when using this? 

Bonus points for a test with the `options` object (see uncovered lines 9-11) 🙂 

Beside that, this looks great!

File: test/index.test.ts [hunk 1]
```diff
@@ -86,6 +95,24 @@ describe('migrate MongoDB state store', () => {
         done();
       });
     });
+
+    it('returns migrations document from custom collection', async done => {
+      const customCollectionName = '__custom_migrations';
+      const customMigrationDoc = { ...migrationDoc, lastRun: `${new Date().getTime()}-my-migration.js` };
```
Comment (dantehemerson) [inline test/index.test.ts:101]: Create a different object of `migrationDoc` defined above to improve test matching differences.

**Comment by qqilihq** [comment]
👍

**Pull request merged**